Sources/SocksCore/TCPSocket.swift

```diff
@@ -104,6 +104,10 @@ public class TCPInternetSocket: InternetSocket, TCPSocket, TCPReadableSocket, TC
 
         try setReuseAddress(true)
     }
+    
+    deinit {
+        try? self.close()
```

Comment:
We can't do much here; we don't even have an error handler. :(
`close()` can only fail for these reasons:
- EBADF: Invalid file descriptor => if this happens, we won't be able to do anything about it
- EINTR: Interrupter by a signal => shouldn't happen unless somebody was using the descriptor outside of `Socks`
- EIO: An I/O error occurred => We can't fix that anyway